linera-explorer/src/lib.rs

```diff
@@ -153,42 +160,42 @@ async fn get_blocks(
     limit: Option<u32>,
 ) -> Result<Vec<Blocks>> {
     let client = reqwest::Client::new();
-    let variables = graphql::blocks::Variables {
+    let variables = blocks::Variables {
         from,
         chain_id,
         limit: limit.map(|x| x.into()),
     };
-    let res = post_graphql::<crate::graphql::Blocks, _>(&client, node, variables).await?;
-    match res.data {
-        None => Err(anyhow!("null blocks data: {:?}", res.errors)),
-        Some(data) => Ok(data.blocks),
-    }
+    Ok(
+        request::<linera_graphql_client::service::Blocks, _>(&client, node, variables)
+            .await?
+            .blocks,
+    )
 }
 
 async fn get_applications(node: &str, chain_id: ChainId) -> Result<Vec<Application>> {
     let client = reqwest::Client::new();
-    let variables = graphql::applications::Variables { chain_id };
-    let result = post_graphql::<crate::graphql::Applications, _>(&client, node, variables).await?;
-    match result.data {
-        None => Err(anyhow!("null applications data: {:?}", result.errors)),
-        Some(data) => Ok(data.applications),
-    }
+    let variables = applications::Variables { chain_id };
+    Ok(
+        request::<linera_graphql_client::service::Applications, _>(&client, node, variables)
+            .await?
+            .applications,
+    )
 }
 
 async fn get_operations(indexer: &str, chain_id: ChainId) -> Result<Vec<Operations>> {
     let client = reqwest::Client::new();
     let operations_indexer = format!("{}/operations", indexer);
-    let variables = graphql::operations::Variables {
+    let variables = operations::Variables {
         from: OperationsKeyKind::Last(chain_id),
         limit: None,
     };
-    let result =
-        post_graphql::<crate::graphql::Operations, _>(&client, &operations_indexer, variables)
-            .await?;
-    result
-        .data
-        .ok_or_else(|| anyhow!("null operations data: {:?}", result.errors))
-        .map(|data| data.operations)
+    Ok(request::<linera_graphql_client::operations::Operations, _>(
```

Comment:
It was not imported yet (it is another operations module, there are a lot of name collisions because of the grahql-client derivation), I named it `gql_operations` as for `services`